specs/interop/predeploys.md

```diff
@@ -323,23 +323,33 @@ chain is included instead.
 
 #### Sending Messages
 
-The following function is used for sending messages between domains:
+The following functions are used for sending messages between domains:
+
+**Permissionless**: Can be relayed on destination chain any address.
 
 ```solidity
 function sendMessage(uint256 _destination, address _target, bytes calldata _message) external returns (bytes32);
 ```
 
-It returns the hash of the message being sent,
-used to track whether the message has successfully been relayed.
-It emits a `SentMessage` event with the necessary metadata to execute when relayed on the destination chain.
+**Permissioned**: Can be relayed on destination chain only by the `_entrypoint` address.
```

Comment:
Maybe 'Permissioned relay'? Or do not mention permissionless and permissioned at all, and simply say that the overloaded can only be relayed through an entrypoint. These words do not sound good in the OP Specs